### PR TITLE
Add example to `optuna.multi_objective.study.create_study`

### PR DESCRIPTION
**Pull request by norihitoishida**
## Motivation
This PR is related to #1334

## Description of the changes
Update docstring, referred to `examples/multi_objective/plot_pareto_front.py` and https://github.com/optuna/optuna/pull/1747 .

### CODE REVIEW

**Review by hvy — APPROVED** [review]
Thanks as always, LGTM!

File: optuna/multi_objective/study.py [hunk 1]
```diff
@@ -48,6 +48,24 @@ def create_study(
...
+                    # Binh and Korn function.
+                    x = trial.suggest_float("x", 0, 5)
+                    y = trial.suggest_float("y", 0, 3)
+
+                    v0 = 4 * x ** 2 + 4 * y ** 2
+                    v1 = (x - 5) ** 2 + (y - 5) ** 2
+                    return v0, v1
+
+                study = optuna.multi_objective.create_study(["minimize", "minimize"])
+                study.optimize(objective, n_trials=3)
```
Comment (HideakiImamura) [inline optuna/multi_objective/study.py:67]: It seems the indent is too deep. Could you reduce them?
```suggestion
    Example:

        .. testcode::

            import optuna

            def objective(trial):
                # Binh and Korn function.
                x = trial.suggest_float("x", 0, 5)
                y = trial.suggest_float("y", 0, 3)

                v0 = 4 * x ** 2 + 4 * y ** 2
                v1 = (x - 5) ** 2 + (y - 5) ** 2
                return v0, v1

            study = optuna.multi_objective.create_study(["minimize", "minimize"])
            study.optimize(objective, n_trials=3)
```

**Review by HideakiImamura — CHANGES_REQUESTED** [review]
Thanks for the PR as always! I have a minor comment.

File: optuna/multi_objective/study.py (hunk 1 above)
Comment (norihitoishida) [reply optuna/multi_objective/study.py]: Thank you for suggestion!

**Review by HideakiImamura — APPROVED** [review]
Thanks for your swift action! LGTM!

**Pull request merged**